<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWidget</class>
 <widget class="QWidget" name="frmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>503</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Fullname :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editFullName"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Occupation :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editOccupation"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="btnSubmit">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
